.github/workflows/codeql-buildscript.sh

```diff
@@ -0,0 +1,5 @@
+#!/usr/bin/env bash
+
+mkdir build && cd build
+cmake ../
+make
```

Comment:
Maybe a stupid question: (Why) is a full build required for a static analyzer?

Reply:
https://docs.github.com/en/code-security/codeql-cli/getting-started-with-the-codeql-cli/preparing-your-code-for-codeql-analysis#creating-databases-for-compiled-languages